app/jobs/html_variant_trial_create_job.rb

```diff
@@ -0,0 +1,7 @@
+class HtmlVariantTrialCreateJob < ApplicationJob
+  queue_as :html_variant_trial_create
+
+  def perform(html_variant_id:, article_id:)
+    HtmlVariantTrial.create(html_variant_id: html_variant_id, article_id: article_id)
```

Comment:
I suggest using `create!` here, in case smth goes wrong, it won't be unnoticed. That's not the topic of this pr to alter the behavior, but still.

Reply:
I have updated to create!